File: math/decimal_to_binary/python/decimal_to_binary.py
def decimal_to_binary(n):
    if n > 1: decimal_to_binary(n//2)
    print(n % 2, end = '')

try: decimal_to_binary(int(input("Enter an Integer to Covert into Binary: ")))
except ValueError: print("Input is not an integer.")
print()

File: math/decimal_to_binary/javascript/decimalToBinary.js
// Main function

function toBinary (input) {
    input = parseInt(input); // Input gets parsed (converts strings into numbers)
    if (Number.isInteger(input)) { // Checks if input is an integer
        return input.toString(2); // Resulting binary gets returned as a string
    }
}

// Converts a decimal integer into binary